packages/cli/src/lib/quire/index.js

```diff
@@ -184,6 +183,74 @@ async function install(version, options={}) {
   await execaCommand('npm install --save-dev')
 }
 
+/**
+ * Install a specific version of `quire-11ty` directly into a quire project
+ *
+ * @param  {String}  projectPath  Absolute system path to the project root
+ * @param  {String}  version  Quire-11ty semantic version
+ * @param  {Object}  options  options passed from `quire new` command
+ * @return  {Promise}
+ */
+async function installInProject(projectPath, version, options={}) {
+  console.debug(`[CLI:quire] installing quire-11ty@${version} into ${projectPath}`)
+
+  /**
+   * delete `package.json` from starter project, as it will be replaced with
+   * `package.json` from `@thegetty/quire-11ty`
+   */
+  await git
+    .cwd(projectPath)
+    .rm(['package.json'])
+    .catch((error) => console.error('[CLI:error] ', error))
+
+  const temp11tyDirectory = '.temp'
```

Comment:
We might want to use a npm package that handles various edge cases around creating the temporary directory. For example if the install to the temp directory fails we need to ensure that we cleanup the file system and notify the user.